quartz_solar_forecast/weather/open_meteo.py

```diff
@@ -0,0 +1,165 @@
+from datetime import datetime
+from typing import List
+
+import pandas as pd
+import requests
+
+
+class WeatherService:
```

Comment:
This version of the `WeatherService` doesn't have the local cache that we developed for training. I think it coud be a useful feature for users to have